games/godot/Portfile

```diff
@@ -37,18 +37,32 @@ if {$subport eq ${name}} {
 }
 
 subport ${name}-3.2 {
+    PortGroup       obsolete 1.0
+
+    replaced_by     ${name}
+    # 2021-12-09: I believe that it would be safe to completely remove
+    # this subport without needing to wait for the traditional one-year
+    # waiting period. The only reason I originally created this subport
+    # was because Godot 3.3+ was failing to compile for macOS <= 10.10.
+    # Now that we have been able to get the source code to compile and
+    # run successfully on older macOSes, this subport is no longer
+    # necessary. In addition, based on the statistics, it looks like
+    # no one has ever installed this subport.
+
     github.setup    godotengine ${name} 3.2.3 "" -stable
 
     checksums       rmd160  4ff4f9c4798d67cf6cf4d63c38bed84d400daa8e \
                     sha256  6cd925cc4c23e2c4b0bf36458a83e03484184d8d5311b8eb2339319df7c0c26c \
                     size    22460848
-
```

Comment:
please also remove the `github.setup` and `checksums`. Additionally I don't think all this commenting is necessary here, I'd just remove all of it and set the date when the subport can be removed which is a year from now (no reason to deviate from that standard).

Reply:
you will need to increase the `revision` of the subport as well to as described in the guide when using the `obsolete` PG

Reply:
> please also remove the `github.setup` and `checksums`.

That was just me being lazy. :smile:

Those comments are mostly only for my benefit, since I'm currently maintaining a couple dozen ports. I suppose I could move those comments into the PR conversation, but it's kind of a hassle to look up old PRs when I'm making changes to my ports.

Reply:
okay, please do still increase the `revision` as requested above

Reply:
Oh, I didn't know I needed to do that. I'll make the change now.

Reply:
Ok, I added the revbump.

Reply:
see: https://guide.macports.org/#development.practices.rename-replace-port